en/mkdocs.yml

```diff
@@ -94,9 +95,10 @@ nav:
       - 'Get Started': guides/before-you-start.md
       - 'Login':
         - 'Web Application':
-          - 'OAuth/OIDC': guides/login/webapp-oidc.md
+          - 'OIDC': guides/login/webapp-oidc.md
           - 'SAML': guides/login/webapp-saml.md
-          - 'STS': guides/login/webapp-sts.md
+          - 'WS-Federation': guides/login/webapp-wsfed.md
+          - 'IWA': guides/login/webapp-iwa.md
         - 'Mobile Application': guides/login/mobile-app.md
```

Comment:
Let's combine Mobile Application and Single Page Application and name it Client-Side Application